Signal/src/util/UIUtil.m

```diff
@@ -46,14 +51,10 @@ + (void)applySignalAppearence
     [[UIToolbar appearance] setTintColor:[UIColor ows_materialBlueColor]];
     [[UIBarButtonItem appearance] setTintColor:[UIColor whiteColor]];
 
-    NSShadow *shadow = [NSShadow new];
-    [shadow setShadowColor:[UIColor clearColor]];
-
-    NSDictionary *navbarTitleTextAttributes = @{
-        NSForegroundColorAttributeName : [UIColor whiteColor],
-        NSShadowAttributeName : shadow,
-    };
-    [[UINavigationBar appearance] setTitleTextAttributes:navbarTitleTextAttributes];
+    // If we set NSShadowAttributeName, the NSForegroundColorAttributeName value is ignored.
```

Comment:
`setTitleTextAttributes` seems to have no effect if this attribute is included, so I don't see an alternative.